conf/references/production_reg_conf.pl

```diff
@@ -77,7 +77,7 @@
 
 # FORMAT: species/alias name => [ host, db_name ]
 my $compara_dbs = {
-    'compara_references' => [ 'mysql-ens-sta-5', 'ensembl_compara_references' ],
+    'compara_references' => [ 'mysql-ens-compara-prod-2', 'ensembl_compara_references' ],
```

Comment:
Just to make sure this is intended (as it is not in the links attached), this is to move back to our domain the run of references, right?